docs/source/user-guide/configuration/use-condarc.rst

```diff
@@ -77,9 +77,15 @@ is available at the terminal or Anaconda Prompt by running
 ``conda config --help``.
 
 .. tip::
+
```

Comment:
@rrigdon Was this whitespace change intentional?

Reply:
Yep.

Reply:
Thanks, just checking -- didn't know if it would affect the rendered content or not.

Reply:
Good looking out. ;)